js/tabs.js

```diff
@@ -430,13 +438,18 @@ Tabs.prototype.readFileToNewTab_ = function(entry, file) {
   reader.readAsText(file);
 }
 
-Tabs.prototype.onSaveAsFileOpen_ = function(tab, close, entry) {
+/**
+ * @param {Tab} tab
```

Comment:
Please be explicit about nullability. Either !Tab or ?Tab :)

Reply:
Done.